cmd/earthly/subcmd/github_cmds.go

```diff
@@ -92,14 +152,97 @@ func (a *Github) actionAdd(cliCtx *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	err = cloudClient.SetGithubToken(cliCtx.Context, a.Org, a.GHOrg, a.GHRepo, a.GHToken)
+	err = cloudClient.CreateGHAIntegration(cliCtx.Context, a.Org, a.GHOrg, a.GHRepo, a.GHToken)
 	if err != nil {
 		return fmt.Errorf("error found running github add: %w", err)
 	}
 	a.cli.Console().Printf("GitHub integration successfully created")
 	return nil
 }
 
+func (a *Github) actionRemove(cliCtx *cli.Context) error {
+	a.cli.SetCommandName("githubRemove")
+	if a.Org == "" {
```

Comment:
this logic appears to be duplicated in multiple locations, can you move it to a function instead? e.g.

```
func (a *Github) getOrgName() { ...
```

or maybe even somehow done under the `before` function: https://github.com/earthly/earthly/blob/1e0e0ef3ea7b9058aaabfab2ad9088b867a22a5b/cmd/earthly/app/before.go#L127-L129

but maybe before won't work since that's used for the top level `--org` flag, and some of our commands have a different `--org` under subcommands, which is confusing and feels like a bigger issue.